Firmware/ultralcd.h

```diff
@@ -47,7 +47,7 @@ void lcd_pause_print();
 void lcd_resume_print();
 void lcd_print_stop();
 void prusa_statistics(int _message, uint8_t _col_nr = 0);
-void lcd_confirm_print();
+//void lcd_confirm_print();
```

Comment:
Please do not comment out code. Just remove it completely if it is not needed.

Reply:
done